ote/src/cljs/ote/style/service_search.cljs

```diff
@@ -107,3 +107,6 @@
 (def contact-icon {:color  "#999999"
                    :height 16
                    :width  16})
+
+(def title-group-description  {:font-size "0.875rem"
+                               :margin-top "-1rem"})
```

Comment:
I think it would be better to just give this `:margin-top 0` and remove the bottom margin from the `heading` element above this text.

Reply:
Heading element is our page/page-controls title. If I change it it will be changed in everywhere. But in other places it should not be done.
So I think that I leave this as it is.